StepDefinitions/Feature1_LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,242 @@
+using Mars_SpecFlowProject1.Pages;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using TechTalk.SpecFlow;
+using Mars_SpecFlowProject1.Utils;
+using NUnit.Framework;
+using SpecFlowProject_MarsProject.Utils;
+using OnboardTaskProjectMars.Pages;
+
+namespace ProjecrMarsOnboardingtask.StepDefinitions
+{
+    [Binding, Scope(Feature = "Feature1_Language")]
+    public class Feature1_LanguageStepDefinitions : HookBase
+    {
+        private readonly LoginPage loginPageObj;
+        private readonly AddLanguageTab languageTabObj;
+        private readonly EditLanguage editLanguageObj;
+        private readonly DeleteLanguage deleteLanguageObj;
+        private readonly AddLanguagewithEmptyName addLanguagewithEmptyNameObj;
```

Comment:
so with your pages this is not done correctly, you should only have skills and langauge page and in those you should have the web elements

Reply:
the way you have broken down your components is not correct! You will learn how to do this properly in the advance task